src/base/contracts/contractInstances.ts

```diff
@@ -16,6 +16,23 @@ export function listContractTypes(abiDirPath: string): string[] {
   });
 }
 
+/**
+ * Return specific ABI as JSON
+ * @param contractName The name of the contract to read the ABI for
+ * @param accessAbiValues Access values to the ABI, some contracts has sub levels to access the ABI, e.g Tron has {"entrys" : [..ABI]}
+ * 
+ */
+export function getJsonAbi(contractName: string, abiDirPath : string, accessAbiValues : string[] = []) {
+  const jsonAbi = fs.readFileSync(path.join( __dirname, `${abiDirPath}/${contractName}.json`), 'utf-8');
+  ensure(isValidJSON(jsonAbi), `Invalid JSON: ${jsonAbi}`);
+  let parsedJsonAbi = JSON.parse(jsonAbi);
+  if (accessAbiValues) {
+    accessAbiValues.forEach(entry => {return parsedJsonAbi = parsedJsonAbi[entry];});
```

Comment:
what exactly is this trying to accomplish? it's weird we have an assignment operator in a return

Reply:
or rather think of it this way - is what this doing clear to a naive developer?

Reply:
I changed this implementation, now should be more clear what this function do. Thanks!